app/views/shared/check_answers/_merits.html.erb

```diff
@@ -28,7 +28,9 @@
   <%= render("shared/check_answers/second_appeal", read_only:) %>
 <% end %>
 
-<%= render("shared/check_answers/matter_opposition", read_only:) if @legal_aid_application.matter_opposition %>
+<%= render("shared/check_answers/matter_opposition", read_only:) if matter_opposition && matter_opposition.reason.present? %>
```

Comment:
Hmm, I don't see a partial for this `check_answers/_matter_opposition`

Reply:
Because it already exists and I didn't change it... only _when_ we display it